### PR TITLE
Completed Calculator Design

### PR DESCRIPTION
**Pull request by AsimKhan2019**
Created the calculator design for the project milestone part2. I used a different layout but the concept is the same. I hope that it will not b an issue.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,40 @@
...
+          <button type="button">0</button>
+          <button type="button">.</button>
+
+          <button type="button">=</button>
+        </div>
+      </div>
+
     </div>
   );
 }
```
Comment (tresorsawasawa) [inline src/App.js]: - [ ] According to the project's requirements, kindly make sure all the `Calculator` code is implemented in the `components/calculator.js` file and not in the `App.js` file. And also make sure it is a component `class-based` and not a `Function-Based`. e.g:
```javascript
import React, { Component } from 'react';

class Calculator extends Component {
  render() {
    // reference as a js object
    return {
      <div>
       // Calculator app code design goes here
      </div>
    };
  }
}

export default Calculator; // export Calculator component
```

- [x] Once you move the above code in your `Calculator.js` file, then make sure you import it(`Calculator.js` file) into the `App.js` main component file and the  `App.js` file should also be used as a **Class-based** component as required for the project and not a **function-based**.

File: src/index.css [hunk 2]
```diff
@@ -1,27 +1,83 @@
...
+.digits {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.digits button {
+  flex: 1 1 33.333%;
+  max-width: 33.333%;
+  background-color: var(--dark);
 }
```
Comment (tresorsawasawa) [inline src/index.css]: - [x] Kindly make sure your design sticks to the provided template. You are supposed to use the same colors and follow the provided styling(each number && symbol at each expected place/position).

| Template Styling | Your Styling  |
| ----------------------- | ------------------- |
| Screenshot from 2022-08-02 21-11-00 | Screenshot from 2022-08-02 21-08-40 |
 #

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @AsimKhan2019 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.css (hunk 2 above)
Comment (AsimKhan2019) [reply src/index.css]: @tresorsawasawa Thank you for the feedback, I have made the changes according to the styles and designs.

File: src/App.js (hunk 1 above)
Comment (AsimKhan2019) [reply src/App.js]: @tresorsawasawa I need help

Comment (AsimKhan2019) [reply src/App.js]: @tresorsawasawa I applied changes to the project as required, I hope that it is correct.

File: src/components/calculator.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+          <button type="button">+</button>
+          <button type="button">-</button>
+          <button type="button">=</button>
+        </div>
+
+        <div className="digits">
+          { createDigits() }
+          <button type="button">0</button>
+          <button type="button">.</button>
+        </div>
```
Comment (uwadonat) [inline src/components/calculator.js]: - [ ] Let's make sure you are following the given design. For now you are using symbols different from the one given on the design. Also check on colors.
- your design:
your_design
- given design:

Screenshot from 2022-08-03 12-48-50

**Review by uwadonat — CHANGES_REQUESTED** [review]
## Hi @AsimKhan2019,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

### To highlight! :+1: 
- Your commit messages are very descriptive ✔️
- Your project is well organized and clean ✔️
- No linter errors  ✔️

 ## Status: Required Changes ♻️

- [ ]  Check the comments under the review.

- [ ] I recommend you to update your readme file with instructions to run your project. It will look more professional.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by rukundoeric — APPROVED** [review]
Hello @AsimKhan2019  👋,

_This is an additional review as a result of the second opinion request._

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


As this was the only reason that your project was not approved it will be approved as soon as possible.
Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approve right away :)

Keep rocking 🚀!

**Review by gemmen29 — APPROVED** [review]
Hi @AsimKhan2019, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AsimKhan2019** [comment]
@gemmen29 Thank you for the approval

**Comment by AsimKhan2019** [comment]
@rukundoeric Thank you for the approval

**Pull request merged**